packages/lwc-services/example/lwc-services.config.js

```diff
@@ -19,14 +19,16 @@ module.exports = {
     sourceDir: './src/client',
     // List of resources for copying to the build folder
     resources: [{ from: 'src/resources', to: 'dist/resources' }],
-    // Default server options for watch command
+    // Default server options for watch command.  See more options at https://webpack.js.org/configuration/dev-server
     devServer: {
         port: 3001,
         host: '0.0.0.0',
         open: false,
         stats: 'errors-only',
         noInfo: true,
         contentBase: './src/client'
+        // run a local backend on another port (ex: nodejs) but still use watch command for lwc live re-loading
```

Comment:
These not, as then we could put basically put more `webpack-dev-server` options into this file, which kinda defeats the purpose.

Reply:
There’s lots of options—but how to do a proxy to a real server seems like a super common use case compared to the other choices.